### PR TITLE
Set default block to legal value

### PR DESCRIPTION
**Pull request by MatzElectronics**
Addresses #107 

To test:  
 - Drag Badge set LED block onto workspace.  
 - Make sure it generates `led(0, 1);`

### CODE REVIEW

**Comment by pjewald** [comment]
Is there an explanation of what was causing the error in #107 and how this PR is correcting that error?

**Comment by MatzElectronics** [comment]
Yes, blocks in the toolbox can be (and usually are) assigned a default value.  The default value assigned to this block was illegal (`high`).  When the user changed the value, it set it to a legal value (`1` or `0`).

File: src/blockly/generators/propcToolbox.js [hunk 1]
```diff
@@ -1012,10 +1012,6 @@ xmlToolbox += '            <category name="Advanced">';
 xmlToolbox += '                <block type="wx_init_adv" exclude="heb-wx,"></block>';
 xmlToolbox += '                <block type="wx_listen">';
 xmlToolbox += '                    <field name="ID">wxConnId1</field>';
-xmlToolbox += '                    <field name="ID1">wxConnId1</field>';
```
Comment (pjewald) [inline src/blockly/generators/propcToolbox.js]: Noting that this commit is unrelated to issue #107.
